js/src/common/components/Modal.js

```diff
@@ -57,7 +70,7 @@ export default class Modal extends Component {
    *
    * @return {Boolean}
    */
-  isDismissible() {
+  static isDismissible() {
```

Comment:
If this is static, why make it a method at all and not just a property?

Reply:
Because it was a method before and allow(ed) for custom logic. :shrug: 

Not sure if that was used anywhere... too lazy to come up with arbitrary use-cases. :laughing:

Reply:
You could perhaps make it dismissible only after an input was filled out and/or form submitted. We can't do that with a static method though, don't have access to the current instance data...

Reply:
Fair point. Should be okay then.